owl/srdl2-cap.owl

```diff
@@ -299,6 +381,14 @@
     
 
 
+    <!-- http://knowrob.org/kb/srdl2-cap.owl#Ear -->
+
+    <owl:Class rdf:about="http://knowrob.org/kb/srdl2-cap.owl#Ear">
```

Comment:
ear is rather a component, so should go into srdl2-comp ontology. same for other components. maybe cap should be imported into comp.